src/logic/tokens/store/actions/fetchSafeTokens.ts

```diff
@@ -38,6 +39,11 @@ const extractDataFromResult = (currentTokens: TokenState) => (
     },
   })
 
+  // Extract network token balance from backend balances
+  if (address === ZERO_ADDRESS) {
```

Comment:
shouldn't we use the `sameAddress` helper?

Reply:
Yes, it's better to use that method